public/humans.txt

```diff
@@ -90,7 +91,11 @@
     Weronika Grzejszczak, Polish
     Wojciech Patelka, Polish
     Carlos Jimenez, Portuguese (Brazil)
+    Carolina Guido, Portuguese (Brazil)
+    Paulo Franco, Portuguese (Brazil)
     Stephan Garcia, Portuguese (Brazil)
+    Urb-i - Urban Ideas, Portuguese (Brazil)
```

Comment:
How was this organization involved?

Reply:
It is the name of the group that Yuval, Carolina, and Paulo are working under. Scroll to the middle of the page here to see more: http://www.urb-i.com/about

Reply:
I see. This is the first time we would have an organization on this list as opposed to individuals so I want to make sure we are okay with setting a precedent for this. The issue I would take here is that this is essentially multiple credit -- all three individuals are credited, but so is their organization. Did they specifically request to be credited in this way? Were all three active and equal collaborators in this?

Could we get away with only crediting the individuals? If the organization must be included, could we include it this way:

```
Carolina Guido (Urb-i), Portuguese (Brazil)
Paulo Franco (Urb-i), Portuguese (Brazil)
Yuval Fogelson (Urb-i), Portuguese (Brazil)
```

If the organization is paramount, could we only credit the organization once? Should it be included in the list of humans or elsewhere?

Lastly, if we do include the org name, I think it would be better to use just the brand (Urb-i) and leave off the slogan.

Reply:
These are good questions, and may come up again at some point. I see your concern.

All three are translators in this, but I am not sure the extent that each person is contributing. Maybe one is contributing more than another person. They are not all the people in their organization.

Where else might the organization be credited (since you make a good point, it's not technically a human)?

I could go back to them and ask if they are ok with this format you're proposing that fits into humans.txt better.

Reply:
I'm going to push back on the name. That's not even how they consistently call themselves on their website. See, for example, their description: 

> Urb-i is composed of a multidisciplinary group of professionals based in São Paulo, Brazil.

Or their copyright at the bottom:

> © 2015-2018 urb-i

If it looks like a tagline, smells like a tagline, then it is a tagline. I'd rather not open that door for everyone else.